.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## *Tips*
+- *Thank you very much for contributing to Apache Hudi.*
+- *Please review https://hudi.apache.org/contributing.html before opening a pull request.*
+
+## What is the purpose of the pull request
+
+*(For example: This pull request adds quick-start document.)*
+
+## Verify this pull request
```

Comment:
How about adding `Brief change log` as flink does?

Reply:
> How about adding `Brief change log` as flink does?

hi, `What is the purpose of the pull request` and `Brief change log` may have similar meaning at a certain exten.

Reply:
Can I suggest adding a "committers checklist"  like below, so the PR author knows what to expect 

- [ ] Has a corresponding JIRA in PR title & commit
- [ ] Commit message is descriptive of the change
- [ ]  CI is green
- [ ] Necessary doc changes done or have another open PR

Reply:
done